migrations/versions/22a0e898310e_add_user_timezone_name.py

```diff
@@ -0,0 +1,57 @@
+"""add_user_timezone_name
+
+Revision ID: 22a0e898310e
+Revises: 2c97929c18d0
+Create Date: 2022-09-01 17:33:48.818171
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "22a0e898310e"
+down_revision = "2c97929c18d0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("user", sa.Column("timezone_name", sa.String(length=255)))
+    op.execute("UPDATE \"user\" SET timezone_name = 'Europe/Paris'")
+    op.execute(
+        """
+      UPDATE \"user\" 
+      SET timezone_name = request.tz 
+      FROM 
+      (
+        SELECT 
+          user_id, 
+          CASE 
+            WHEN cp = '984' THEN 'Indian/Kerguelen'
+            WHEN cp = '986' THEN 'Pacific/Wallis'
+            WHEN cp = '987' THEN 'Pacific/Tahiti'
+            WHEN cp = '988' THEN 'Pacific/Noumea'
+            WHEN cp = '971' THEN 'America/Guadeloupe'
+            WHEN cp = '972' THEN 'America/Martinique'
+            WHEN cp = '973' THEN 'America/Guyana'
```

Comment:
:red_circle: La timezone qui correspond à la Guyane Française est America/Cayenne .
```suggestion
            WHEN cp = '973' THEN 'America/Cayenne'
```